core/arch/arm/plat-imx/a9_plat_init.S

```diff
@@ -88,7 +88,7 @@ UNWIND(	.fnstart)
 	 * PCR
 	 * - no change latency, enable clk gating
 	 */
-	mov_imm r0, 0x00004000
+	mov_imm r0, 0x00004800
```

Comment:
Aha, so we're setting a few bits in this register in `generic_entry_a32.S` and first thing here is to just override everything, seems broken. Both `SCTLR_RR` and `SCTLR_Z` should probably be set in `generic_entry_a32.S` instead. or am I missing something?